tensorflow/core/BUILD

```diff
@@ -2772,6 +2772,7 @@ cc_library(
     visibility = ["//visibility:public"],
     deps = [
         "//tensorflow/core/platform/default/build_config:stream_executor_no_cuda",
+        "//tensorflow/stream_executor:rocm_platform"
```

Comment:
a comment here is needed. this dependency would likely need to be removed, and it's `stream_executor_no_cuda` to be amended.